package.json

```diff
@@ -31,11 +31,10 @@
     "@newrelic/test-utilities": "^4.1.2",
     "apollo-server": "^2.18.2",
     "eslint": "^7.8.1",
-    "newrelic": "^6.13.0",
+    "newrelic": "^7.3.1",
```

Comment:
This works fine but could also be `>=6.13.0`.

Reply:
okay!